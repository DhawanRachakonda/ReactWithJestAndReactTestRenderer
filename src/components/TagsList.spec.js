import React from 'react';
import renderer from 'react-test-renderer';

import TagsList from './TagsList';

describe('tests snapshot for TagsList', ()=>{
    it('checks the generated shapshot tree', ()=>{
        const tree = renderer.create(<TagsList tags={['python', 'ruby', 'java']}/>);
        tree.toJSON();
        expect(tree).toMatchSnapshot();
    });
})