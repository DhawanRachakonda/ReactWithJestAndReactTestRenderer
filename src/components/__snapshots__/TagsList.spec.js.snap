// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests snapshot for TagsList checks the generated shapshot tree 1`] = `
<div>
  <code>
    python
  </code>
  <code>
    ruby
  </code>
  <code>
    java
  </code>
</div>
`;
